Mathlib/Topology/Algebra/ContinuousMonoidHom.lean

```diff
@@ -391,3 +391,43 @@ theorem locallyCompactSpace_of_hasBasis (V : ℕ → Set Y)
 end LocallyCompact
 
 end ContinuousMonoidHom
+
+section
+
+universe u v
+
+variable (G : Type u) [Group G] [TopologicalSpace G] [TopologicalGroup G] (H : Type v)
+  [Group H] [TopologicalSpace H] [TopologicalGroup H]
+
+/-- Define the structure of two-sided continuous isomorphism. -/
+structure ContinuousMulEquiv extends MulEquiv G H , Homeomorph G H
```

Comment:
I think that this structure needs a proper API (e.g. `ContinuousMulEquivClass`, `ext`, `refl`, `copy`...) and should probably be its own PR.

Reply:
Maybe I can start a new PR to solve this. So where can I learn from to write these APIs?

Reply:
I agree with @kbuzzard that some API is needed (you can have a look at lines 136-186 of the file Mathlib/Algebra/Group/Equiv/Basic.lean where they are defined); but I think that even before that you should consider extending the `MulEquiv` class as explained in line 100 there.

Reply:
Thank you, I would start a new PR as soon as I am finish the PRs for Profinite

Reply:
So you are going to put this on hold?

Reply:
Don't forget an additive version of this

Reply:
I would try to do this as soon as possible, right after fixing this PR and making the old proofs to fit the new definition (there are like thousand lines depending on the limit of profinite group)